test/api/fragment.js

```diff
@@ -18,11 +18,10 @@ describe("API: JSDOM.fragment()", () => {
     assert.strictEqual(frag1.ownerDocument, frag2.ownerDocument);
   });
 
-  it("should return fragments with shared Windows each time", () => {
-    const frag1 = JSDOM.fragment(``);
-    const frag2 = JSDOM.fragment(``);
+  it("should return a fragment with no associated browsing context", () => {
+    const frag = JSDOM.fragment(``);
 
-    assert.strictEqual(frag1.ownerDocument.defaultView, frag2.ownerDocument.defaultView);
+    assert.isNull(frag.ownerDocument.defaultView);
```

Comment:
>  E.g. XHR shouldn't work, if we've implemented it right.

@domenic I don't know exactly how you would like to test XHR since the globals are not accessible via the `defaultView`.

Reply:
Oh, I actually looked at the diff, and I see that of course, if `defaultView` is null, then you wouldn't be able to get an XHR.

Reply:
> The defaultView IDL attribute of the Document interface, on getting, must return this Document's browsing context's WindowProxy object, if this Document has an associated browsing context, or null otherwise.

https://html.spec.whatwg.org/#dom-document-defaultview

Reply:
Based on this, it think it worth adding this change for the next major version, since it's a real breaking change. 

I did a search on github to see the usage of the `JSDOM.fragment` API, non of the code that I have seen try to access the `defaultView` out of the fragment. Better be safe than sorry.